spec/index.bs

```diff
@@ -9640,6 +9781,25 @@ The following enums are supported if and only if the {{GPUFeatureName/"depth32fl
         * {{GPUTextureFormat/"depth32float-stencil8"}}
 </dl>
 
+## <dfn dfn-type=enum-value dfn-for=GPUFeatureName>"multi-draw-indirect"</dfn> ## {#multi-draw-indirect}
+
+Allows the use of the {{GPURenderEncoderBase/multiDrawIndirect()}} and
+{{GPURenderEncoderBase/multiDrawIndexedIndirect()}} methods and removes the zero value restriction on
```

Comment:
We have "indirect-first-instance" feature to remove the `firstInstance` restriction. Why does this other feature mention this?

Reply:
Good point. Could we specify this feature without touching the logic covered by  "indirect-first-instance"?
I.e. the new methods you are introducing would just not be a subject to that old feature, but it doesn't mean that all the other methods should be affected by your feature.

Reply:
removed reference to `firstInstance` restrictions